official/resnet/resnet_run_loop.py

```diff
@@ -168,8 +168,14 @@ def learning_rate_with_decay(
   vals = [initial_learning_rate * decay for decay in decay_rates]
 
   def learning_rate_fn(global_step):
+    """Builds scaled learning rate function with 5 epoch warm up."""
     global_step = tf.cast(global_step, tf.int32)
```

Comment:
I don't know if it matters, but I imagine global_step is either an int or a float at the outset, so if we cast to int here, we can avoid the second cast to float in line 176 if we just preserve the original global_step. Or vice versa. Not sure how much that additional cast would matter in this case.